### PR TITLE
player buckets fix

### PR DESCRIPTION
**Pull request by DevX32**
## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by FjamZoo — APPROVED** [review]

File: server/events.lua [hunk 1]
```diff
@@ -1,5 +1,5 @@
 -- Event Handler
-
+QBX.Player_Buckets = {}
```
Comment (Manason) [inline server/events.lua:2]: These are already defined here: https://github.com/Qbox-project/qbx_core/blob/main/server/main.lua#L15

Why do they need to be set here as well?

**Review by Manason — CHANGES_REQUESTED** [review]

File: server/events.lua (hunk 1 above)
Comment (DevX32) [reply server/events.lua:2]: when player drops error pop ups player_buckets nil

Comment (Manason) [reply server/events.lua:2]: I'm afraid I don't understand.

Comment (DevX32) [reply server/events.lua:2]: if u are entering the server and u exited or something u get and error called playerbuckets nil

Comment (Manason) [reply server/events.lua:2]: This doesn't seem like the right solution as one of the two QBX.Player_Buckets = {} may overwrite existing data and it's confusing to clear it twice. Rather instead I would suggest checking that QBX.Player_Buckets exists before trying to clear the player out of one

**Review by Manason — CHANGES_REQUESTED** [review]

File: server/events.lua (hunk 1 above)
Comment (DevX32) [reply server/events.lua:2]: i will  try to find the perma and right solution

**Pull request closed without merging**